### DISCUSSION TITLE
Is there any methods can inactive the drawingtool to implement a browsing mode

### DISCUSSION THREAD

**Discussion by Songhao-LI** [annotorious/annotorious#451] [Q&A]
Hi all,

I hope to implement a browsing mode in my Reace project with annotorious 2.x. (in this mode, users can't start to draw annotation even if they are holding SHIFT button.)

 Like this: https://annotorious.dev/getting-started/
![IMG_5087](https://github.com/user-attachments/assets/044e769a-0a50-487a-9c11-5e4f680a6ad1)

But i don't find any api about this. 🥲🥲🥲
I will be very grateful if anyone can tell me how to do!!

**Comment by rsimon** [comment]
I think you might be looking for the `readOnly` init option. See old v2 docs here:
https://annotorious.github.io/api-docs/osd-plugin/

I believe this init option should also disable drawing altogether.
